packages/web-components/src/components/va-modal/va-modal.scss

```diff
@@ -62,13 +68,19 @@
   background-color: var(--color-green-lightest);
 }
 
-:host([status='success']) .usa-modal::before {
+:host([status='success']) .usa-modal__main::before {
   color: var(--color-gray-dark);
   content: '\F00C'; /* fa-check */
 }
 
-.usa-modal::before {
+.usa-modal__main::before {
   padding-top: 0;
+  position: absolute;
+  left: -1.6rem;
```

Comment:
Does this still position ok on mobile viewports?

Reply:
Yes, I moved the psuedo element was to be a child of `.usa-modal__main` which is the content wrapper. This prevents the spacing between the icon and the title from changing.

![Screenshot 2024-02-14 at 1 56 15 PM](https://github.com/department-of-veterans-affairs/component-library/assets/1776069/8cbd6fc8-8a0d-4984-8058-01d44738d145)